packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -404,16 +400,18 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
           <FluentChatMessage
             attached={attached}
             key={props.message.messageId}
-            root={{ className: chatMessageStyles.root }}
-            author={<Text className={chatMessageAuthorStyle}>{message.senderDisplayName}</Text>}
-            body={{
-              className: chatItemMessageContainerClassName,
-              style: { ...createStyleFromV8Style(messageContainerStyle) },
+            root={{
+              className: chatMessageStyles.root,
               // make body not focusable to remove repetitions from narrators.
               // inner components are already focusable
               tabIndex: -1,
               role: 'none'
             }}
+            author={<Text className={chatMessageAuthorStyle}>{message.senderDisplayName}</Text>}
```

Comment:
Nit: it would be nice to have these alphabetized, or at least the same line as the previous version so comparing is easier.

Reply:
As discussed, will leave as it is for now and we can review this later (possible with adding a rule to linter)